app/site/controllers.py

```diff
@@ -52,6 +51,7 @@ def logout():
       302:
         description: Load the Home Page
     """
+    g.current_user = None
```

Comment:
Why do we need to set this here? Surely once we've expired the cookie it won't get set on the next request.